lib/ios/reactnativeuilib/keyboardtrackingview/KeyboardTrackingViewTempManager.m

```diff
@@ -505,30 +505,11 @@ - (void) rctContentDidAppearNotification:(NSNotification*)notification
     });
 }
 
-#pragma mark - root view controller and presenting style
-
-
-- (BOOL)isModal {
-    UIWindow *window = UIApplication.sharedApplication.windows.firstObject;
-    UIViewController *presentedViewController = window.rootViewController.presentedViewController;
-
-    if (presentedViewController != nil) {
-        if ([presentedViewController isKindOfClass:[UINavigationController class]]) {
-            UINavigationController *presented = (UINavigationController*)presentedViewController;
-            if (presented.visibleViewController) {
-                return YES;
-            }
-        }
-    }
-    return NO;
-}
-
```

Comment:
Do you know what was that for?

Reply:
Yes. I wrote it and forgot to remove it after I moved to another solution